gm4_block_compressors/data/gm4_block_compressors/functions/item/decompress.mcfunction

```diff
@@ -12,5 +12,6 @@ execute if entity @s[nbt={Item:{tag:{gm4_precompression_tag:"no_tag_data_to_pres
 
 #restore original item tag, overwriting tags from compression
 data modify entity @s Item.tag set from entity @s Item.tag.gm4_precompression_tag
+data merge entity @s {PickupDelay:4}
```

Comment:
Note to self: Check whether `merge` or `modify` is faster for constants.

Reply:
I've tested it:
`modify`: AVG tps: 329
`merge`: AVG tps: 283.6

`merge` is worse. I suspect it's because when you do merge, it has to parse the entire nbt to check if it's already set to that, but modify only checks that specific nbt tag?